js/summary.go

```diff
@@ -21,21 +21,31 @@
 package js
 
 import (
-	"bytes"
+	_ "embed" // this is used to embed the contents of summary.js
 	"fmt"
 	"io"
+	"strings"
 	"time"
 
 	"github.com/dop251/goja"
 	"go.k6.io/k6/js/common"
 	"go.k6.io/k6/lib"
 	"go.k6.io/k6/stats"
-	"go.k6.io/k6/ui"
 )
 
-// TODO: move this to a separate JS file and use go.rice to embed it
-const summaryWrapperLambdaCode = `
+// Copied from https://github.com/k6io/jslib.k6.io/tree/master/lib/k6-summary
```

Comment:
This means we can deprecate the jslib, right?

Reply:
No? jslib has a ton of other libraries. Besides, having `textSummary()` in there is very helpful for users that want to define their own `handleSummary()` function, e.g. to emit a json or a junit file, but they also want to continue getting the text summary at the end of the test. They can just `import { textSummary } from 'https://jslib.k6.io/lib/k6-summary/0.0.1/index.js'` and use that in `handleSummary()`'s result.

Reply:
I didn't mean deprecating jslib.k6.io altogether :smile:, just this lib.

I wasn't aware there were differences between the version committed here and the one on jslib. Though since we already bundle it in the binary, couldn't we expose it as a built-in JS module and avoid the remote loading? :thinking:

Reply:
There aren't any differences, except we don't bundle the JUnit code (we'll probably split it apart in jslib as well soon). And we can potentially expose it as a JS module, if we figure out a nice module name, but that will be a separate issue/PR